source/style-guide/style/telephone-numbers.txt

```diff
@@ -22,7 +20,7 @@ number.
    * - 1 800 961 4454
      - 1 (800) 961-4454
 
-Precede US and Canadian telephone numbers with ``+1``. Precede all
+Precede US and Canadian telephone numbers with ``1``. Precede all
 other telephone numbers with a plus sign.
```

Comment:
**issue:** This is important: Non-US or Canadian users would use `+1` and many of our customers are outside of North America.

[Twilio](https://support.twilio.com/hc/en-us/articles/223183008-Formatting-International-Phone-Numbers), a company that lives off phone numbers, uses the [E.164 standard](https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-E.164-201011-I!!PDF-E&type=items). Twilio [explains the format](https://www.twilio.com/docs/glossary/what-e164).

```suggestion
Precede US and Canadian telephone numbers with ``+1``. Precede all
other telephone numbers with a plus sign.
```

Reply:
I definitely misunderstood what this text was getting at, and the example following it is wrong too. Updated everything.